docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,71 @@
+
+### Writing New Series Asynchronously
+
+The default M3DB YAML configuration will contain the following as a top-level key under the `db` section:
+
+```
+writeNewSeriesAsync: true
+```
+
+This instructs M3DB to handle writes for new timeseries (for a given time block) asynchronously. Creating a new timeseries in memory is much more expensive than simply appending a new write to an existing series, so the default configuration of creating them asynchronously improves M3DBs write throughput significantly when many new series are being created all at once.
+
+However, since new time series are created asynchronously, its possible that there may be a brief delay inbetween when a write is acknowledged by the client and when that series becomes available for subsequent reads.
+
+M3DB also allows operators to ratelimit the number of new series that can be created per second via the following configuration:
+
+```
+writeNewSeriesLimitPerSecond: 1048576
+```
+
+This value can be set much lower than the default value for workloads in which a significant increase in cardinality usually indicates an abusive caller.
+
+### Ignoring Corrupt Commitlogs on Bootstrap
+
+If M3DB is shutdown gracefully via a sigterm signal, it will ensure that all pending writes are flushed to the commitlog on disk before the process exists. However, in situations where the process crashed/exited unexpectedly or the node itself experienced a sudden failure, the tail end of the commitlog may be corrupt.
+
+In that situation M3DB will read as much of the commitlog as its able to in order to recover as much data as possible, however it then needs to make a decision. It can either come up successfully and tolerate the minor amount of data loss or it can attempt to stream the missing data from its peers. This behavior is controlled by the following default configuration:
+
+```
+bootstrap:
+	commitlog:
+		returnUnfulfilledForCorruptCommitLogFiles: false
+```
+
+In the situation where only a single node fails, the optimal outcome is for the node to attempt to repair itself from one of its peers, however, if a quorum of nodes fail and encounter corrupt commitlog files then they will get stuck in a deadlock attempting to stream data from each other with none of them able to make any progress due to the fact that they cannot achieve quorum. This issue requires an operator with significant M3DB operational experience to step in and resolve, so we recommend running with the default configuration set to `false` to avoid this issue altogether since in most cases a small amount of data loss is preferable to a quorum of nodes that crash and fail to start back up automatically.
```

Comment:
nits:

> attempt to repair itself from one of its peers. However, [...]

> and encounter corrupt commit log files, they will deadlock while attempting to stream data from each other, as no nodes will be able to make progress due to a lack of quorum.

> an operator with significant M3DB operational experience to manually bootstrap the cluster; thus, the official recommendation is to set `returnUnfulfilledForCorruptCommitLogFiles: false` to avoid this issue altogether. In most cases, a small amount of data loss [...]